contracts/FleekERC721.sol

```diff
@@ -271,6 +278,46 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
         emit NewTokenDescription(tokenId, _tokenDescription, msg.sender);
     }
 
+    /**
+     * @dev Updates the `logo` metadata field of a minted `tokenId`.
+     *
+     * May emit a {NewTokenLogo} event.
+     *
+     * Requirements:
+     *
+     * - the tokenId must be minted and valid.
+     * - the sender must have the `tokenController` role.
+     *
+     */
+    function setTokenLogo(
+        uint256 tokenId,
+        string calldata _tokenLogo
```

Comment:
Maybe we add color to this input as well and make it optional and call the underlying setTokenColor so they can do it one tx like mint. Not required but just thinking out loud